### PR TITLE
Refactor TesteeModelTest and TesteesServiceTest

### PR DESCRIPTION
**Pull request by Yarik2308**
https://github.com/malinink/easy-tests/issues/261

- [x] TesteeModelTest
- [x] TesteesServiceTest

### CODE REVIEW

**Comment by Yarik2308** [comment]
@malinink seems like everything is done.

File: src/test/java/easytests/core/services/TesteesServiceTest.java [hunk 1]
```diff
@@ -7,14 +7,16 @@
...
-import easytests.support.Models;
+import easytests.support.QuizzesSupport;
+import easytests.support.TesteesSupport;
 import org.junit.Assert;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import static org.mockito.BDDMockito.*;
+
```
Comment (malinink) [inline src/test/java/easytests/core/services/TesteesServiceTest.java]: Remove empty line

File: src/test/java/easytests/support/TesteesSupport.java [hunk 2]
```diff
@@ -1,11 +1,18 @@
 package easytests.support;
 
 import easytests.core.entities.TesteeEntity;
+import easytests.core.models.SubjectModel;
+import easytests.core.models.TesteeModel;
 import easytests.core.models.TesteeModelInterface;
 import easytests.core.models.empty.QuizModelEmpty;
 import org.junit.Assert;
 import org.mockito.Mockito;
```
Comment (malinink) [inline src/test/java/easytests/support/TesteesSupport.java]: also remove empty line

**Comment by Yarik2308** [comment]
@malinink Everything is done.

**Comment by malinink** [comment]
@Yarik2308 I also found that you did not change map function in service, there is an old [cs] bug with ArrayList creation, fix it pls!

**Comment by Yarik2308** [comment]
@malinink If I get you right, where was a mistake in 'ArrayList<>'.

**Pull request merged**